website/docs/docs/build/metricflow-commands.md

```diff
@@ -405,9 +408,8 @@ mf query --metrics order_total --group-by order_id__is_food_order --limit 10 --o
 | 2017-08-29    | False           |         333.65 |
 | 2017-08-28    | False           |         334.73 |
 ```
-</TabItem>
 
-<TabItem value="eg4" label="where clause">
+### Add where clause
 
 You can further filter the data set by adding a `where` clause to your query. The following example shows you how to query the `order_total` metric, grouped by `is_food_order` with multiple where statements (orders that are food orders and orders from the week starting on or after Feb 1st, 2024). Note that when you query a dimension, you need to specify the primary entity for that dimension. In the following example, the primary entity is `order_id`.
 
```

Comment:
One more issue here - the `metric_time__week` part of the filter won't work. Here's what the full where filter should look like:
`--where "{{ Dimension('order_id__is_food_order') }} = True and {{ TimeDimension('metric_time', 'week') }} >= '2024-02-01'"`